### DISCUSSION TITLE
What is the difference between hover:prose-a:underline and prose-a:hover:underline?

### DISCUSSION THREAD

**Discussion by isaacfink** [tailwindlabs/tailwindcss#13366] [Help]
I can see the generated css is different, the hover is applied to the parent in the second example (`prose-a:hover:underline`) wouldn't it make more sense if it was the other way around? in my mind `prose-a:hover` means hover on `prose-a` (which is the a tag)

**Comment by isaacfink** [comment]
The second way feels more similar to CSS

**Comment by adamwathan** [comment]
Hey! This is explained in the docs here:

https://tailwindcss.com/docs/hover-focus-and-other-states#ordering-stacked-modifiers

Agree it's sort of unintuitive right now, might revisit it for Tailwind 4 but would be a pretty rough breaking change too so will have to weight it.

**Comment by isaacfink** [comment]
@adamwathan I am trying to do some more open source so I wanna take a stab at this, is it part of tailwind or the plugin?
